Modules/OrgUnit/classes/ARHelper/RowActions.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+namespace ILIAS\Modules\OrgUnit\ARHelper;
+
+class RowActions
```

Comment:
Please rename to "DropdownBuilder" and add a little context in the comment. I guess we would want to have an interface like this one on the `Dropdown` itself. Please propose an according entry to the roadmap of the UI framework in a separate PR.